provisions/roles/jenkins/master/tasks/install.yml

```diff
@@ -15,5 +15,9 @@
 - name: Ensure Jenkins is installed.
   yum: name=jenkins-2.67-1.1 state=present
 
+- name: Reload jenkins systemd service
+  sudo: yes
+  service: name=jenkins state=reloaded
```

Comment:
Based on https://github.com/ansible/ansible-modules-core/issues/3897#issuecomment-224893624, I'd recommend we use `systemd` module instead of `service` module.